cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,31 @@
+const { generateUser } = require('../support/generate');
+
 describe('Sign Up page', () => {
+  beforeEach(() => {
+    cy.visit('https://react-redux.realworld.io')
+  });
+
   it('should provide an ability to register new account', () => {
+    const { email, password, username} = generateUser()
+
+    cy.get(':nth-child(3) > .nav-link')
+      .should('contain.text', 'Sign up')
+      .click();
+    
+    cy.get('[placeholder="Username"]')
```

Comment:
It'll be cool to write the `findPlaceholder` function in `commands.js` but OK this way